.github/workflows/snapshot.yml

```diff
@@ -0,0 +1,29 @@
+name: Snapshot
+
+on: workflow_dispatch
+
+jobs:
+  release:
+    name: Publish snapshot version
+    runs-on: ubuntu-latest
+    env:
+      CI: true
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up Node.js 12.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 12.x
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Publish
+        uses: seek-oss/changesets-snapshot@v0
```

Comment:
Should we release a `v1` branch now that it's widely used?

Reply:
I guess we could. Feels weird to go to v1 just on API stability though, rather than new features.